test/e2e/serial/tests/workload_placement.go

```diff
@@ -246,6 +243,9 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 			By("wait for NRT data to settle")
 			e2efixture.MustSettleNRT(fxt)
 
+			nrtPostCreateDeploymentList, err := e2enrt.GetUpdated(fxt.Client, nrtInitialList, time.Minute)
+			Expect(err).ToNot(HaveOccurred())
+
```

Comment:
we had a similar bug in a different test, didn't we?

Reply:
yes indeed, fixed below, but hard to catch since it isn't always failing:
https://github.com/openshift-kni/numaresources-operator/commit/129621266defbcb71483a8d39e1b8c3826428ed7